### DISCUSSION TITLE
How to suggest a new feature?

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#275] [Discuss: New Feature Ideas]

**Comment by phoneticallySAARTHaK** [comment]
How do I **create** a new feature request?
Do I have to create a new discussion first?

The feature request option in issues tab takes to the "suggest-new-feature" discussions (which doesn't exist)  

-----
![Screenshot_20221216-000418](https://user-images.githubusercontent.com/71564671/207940991-dcc7e98a-199d-4599-be38-bc67b73676e7.png)

-----

**Reply by cruxcode** [reply]
Yes @phoneticallySAARTHaK. We will convert that discussion into feature request once we have a small discussion on it.
